packages/api/src/schema/schema.yaml

```diff
@@ -696,8 +695,9 @@ components:
       type: object
       additionalProperties: false
       properties:
-        status:
-          $ref: "#/components/schemas/webhook/properties/status"
+        errorMessage:
+          type: string
+          description: Error message if the webhook failed to process the event
         response:
           $ref: "#/components/schemas/webhook-response"
```

Comment:
Hmm... Maybe here we actually want to send a `"#/components/schemas/webhook-response/properties/response"` instead? seems like this objet has a lot of other fields that we don't need while we only really use the `response.response` field anyway (there is a status code in there too). Would simplify the client and the server-side code. This is the most optional change.